src/platform/packages/shared/kbn-es-query/index.ts

```diff
@@ -39,6 +39,8 @@ export type {
   ScriptedPhraseFilter,
   ScriptedRangeFilter,
   TimeRange,
+  TimeState,
+  RelativeTimeRange,
```

Comment:
Should we export `AbsoluteTimeRange` too? That might be the type missing an export in the build stats.

Reply:
ah, sure thing, thanks